figures/settings.py

```diff
@@ -12,6 +15,16 @@
 
 DAILY_METRICS_CELERY_TASK_LABEL = 'figures-populate-daily-metrics'
 
+DEFAULT_LOG_PIPELINE_ERRORS_TO_DB = True
+
+env_tokens = {}
+
+
+def log_pipeline_errors_to_db():
+    return True if env_tokens.get(
+        'LOG_PIPELINE_ERRORS_TO_DB',
+        DEFAULT_LOG_PIPELINE_ERRORS_TO_DB) else False
```

Comment:
->

```py
return env_tokens.get('LOG_PIPELINE_ERRORS_TO_DB', DEFAULT_LOG_PIPELINE_ERRORS_TO_DB)
```
There seems to be no need for the `if` statement.

Reply:
There is no guarantee that the lms.env.json setting is going to be a boolean value. This forces it to be

Reply:
The type in this case doesn't really matter. What matters is if the value is truthy or falsey. But if you insist on returning a boolean type, the I'd like to suggest the following form:

```py
def log_pipeline_errors_to_db():
    should_log = env_tokens.get('LOG_PIPELINE_ERRORS_TO_DB', DEFAULT_LOG_PIPELINE_ERRORS_TO_DB)
    return bool(should_log)
```

There's no branching and the function is much easier to read.

Reply:
sure